en/flight_modes/land.md

```diff
@@ -0,0 +1,43 @@
+# LAND Flight Mode
+
+The LAND flight mode causes the vehicle to land at the position where the mode was engaged.
+
+The specific behaviour for each vehicle type is described below.
+
+> **Note** 
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+
+## Multi-Copter (MC)
+
+The vehicle will land at the location at which the mode was engaged. The vehicle descends at the rate specified in `MPC_LAND_SPEED` until it hits the ground.
+
+Landing is affected by the following parameters:
+
+Parameter | Description
+--- | ---
+[MPC_LAND_SPEED](../advanced_config/parameter_reference.md#MPC_LAND_SPEED) | The rate of descent during landing. This should be kept fairly low as the ground conditions are not known.
+
+## Fixed Wing (FW)
+
+The vehicle turns and lands at the location at which the mode was engaged. 
+
+> **Note** The vehicle doesn't necessarily know where the ground is, and will therefore follow a fixed trajectory for the whole landing approach (no flare).
+
+Landing is affected by the following parameters:

```

Comment:
@dagar 

> It seems like we should be explaining FW LAND logic somewhere, and then reference it for LAND, MISSION, and RTL.

Probably. But I'm still working out the logic. I will ping you on slack if this isn't right. Promise :-)

> This is where it gets convoluted. The autopilot likely doesn't know the actual height of the ground. When you jump into LAND as a mode it's the same landing behavior, but it internally created the land waypoint at altitude 0. As a result you probably aren't going to hit the flare logic (FW_LAND_FLALT) unless you actually get to FW_LAND_FLALT over the land altitude. If you have a distance sensor (and it's enabled), then it will be used instead.

So what I think you are saying is that (please confirm):
1. Landing logic is always the same/similar - e.g. for Mission, LAND and RTL, but there are different assumptions about where ground is in each case, and because of that flaring may or may not happen.
2. FW landing has multiple phases. The first phase is basically a fixed trajectory(FW_LND_ANG)  towards the ground. At a certain height (FW_LAND_FLALT) above the specified land altitude you'll try to flare [as per this diagram](https://github.com/PX4/Firmware/blob/master/Documentation/fw_landing.png). 
- for LAND mode you set the land altitude at 0 (sea level). Because the land is likely to be significantly above sea level you never descend to FW_LAND_FLALT so you never engage the flare logic - just proceed on your initial fixed trajectory. 
  - If you have a distance sensor this will be used. At that point the landing will do the proper flaring, because you know where land is.
- FOR RTL, you may not land because the default behaviour is to circle. However if you do land, you will use the HOME position as your landing altitude. That should be correct because you know when you took off what the height was. As a result the flaring will be above this point, and will be engaged.
- FOR Mission, you specify the height of your end landing point (????) so that will be the point that is used for ground. A proper landing with flare should happen. Correct?

So, what parameter is used to specify the LAND ALTITUDE - from which FW_LAND_FLALT is calculated.

IF THE ABOVE IS CORRECT, then I'm nearly good to document this :-) What would be great from you  [a version of  this diagram](https://github.com/PX4/Firmware/blob/master/Documentation/fw_landing.png) labeled with our parameters. 

On that note, can you confirm this definition of FW_LND_HVIRT:
```
FW_LND_HVIRT - Virtual horizontal line/altitude used to calculate the flare trajectory.

The FW_LND_HVIRT represents the notional sub-ground altitude where the vehicle would reach the desired post-flare touchdown position, if it were to maintain its initial descent trajectory. This negative altitude (expressed as a positive number) is used as part of the algorithm for calculating the flare path. 
```
If correct, then the diagram should also show the intersection of the FW_LND_HVIRT and the touchdown position.